cypress/integration/tests/administrator/repository/git.test.ts

```diff
@@ -102,25 +101,24 @@ describe("Test an application form a Git source", { tags: "@tier1" }, () => {
         application.openreport();
     });
 
-    it("Disable insecure git Repository", () => {
+    it("Analysis on insecure git Repository(http) for tackle test app", function () {
```

Comment:
Both the tests have same name .

it("Analysis on insecure git Repository(http) for tackle test app when insecure repository is allowed", function () {

it("Analysis on insecure git Repository(http) for tackle test app when insecure repository not allowed", function () {